alembic/versions/c889c45342e7_split_application_table.py

```diff
@@ -0,0 +1,64 @@
+"""split application table
+
+Revision ID: c889c45342e7
+Revises: 9152eefe06e0
+Create Date: 2023-08-14 10:53:58.073719
+
+"""
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSON, UUID
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "c889c45342e7"
+down_revision: Union[str, None] = "9152eefe06e0"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+# I have gonbe the nuclear option of dropping and recreating the table
+# as we have no valuable data at this stage and it is significantly easier
+# than trying to build a migration script.
+def upgrade() -> None:
```

Comment:
Do we need to be adding a FK constraint within this upgrade?

Reply:
added